app/templates/main/index.html

```diff
@@ -3,13 +3,24 @@
 {% block content %}
     <link rel="stylesheet" type="text/css" href="static/styles/map.css">
     <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAhkKJDnPxq0_xvF6pLAFcKfVAL6GbARBs&libraries=places&callback=initMap"
+        async defer></script>    
```

Comment:
can you make 'async' to start on the same column as 'src'? And </script> on a new line, same column as <script>